cmd/cloud/deployment.go

```diff
@@ -360,3 +371,50 @@ func deploymentVariableUpdate(cmd *cobra.Command, args []string, out io.Writer)
 
 	return deployment.VariableModify(deploymentID, variableKey, variableValue, ws, envFile, variableList, useEnvFile, makeSecret, true, astroClient, out)
 }
+
+func newDeploymentWorkerQueueRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "worker-queue",
+		Aliases: []string{"wq"},
+		Short:   "Manage deployment worker queues",
+		Long:    "Manage worker queues for an Astro Deployment.",
+	}
+	cmd.AddCommand(
+		newDeploymentWorkerQueueCreateCmd(out),
+	)
+	return cmd
+}
+
+func newDeploymentWorkerQueueCreateCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "create",
+		Aliases: []string{"cr"},
+		Short:   "Create a Deployment's worker queue",
+		Long:    "Create a worker queue for an Astro Deployment",
+		Example: "", // TODO should we add some examples for default and custom worker-queue commands?
```

Comment:
Would a user add a default worker queue. I thought the default worker queue already existed. If they made changes to it wouldn't that be an update?

Reply:
There can only be one default worker queue, for which we can't rename (but can update other configurations). That would be an update yep

Reply:
removed the `--is-default` flag as it was not needed. Users should not be able to create a default worker queue.